app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -160,6 +160,7 @@ public boolean onContextItemSelected(MenuItem item) {
 
     public void setItems(List<NavDrawerData.DrawerItem> listItems) {
         this.listItems = listItems;
+        notifyDataSetChanged();
```

Comment:
notifyDataSetChanged is already called from outside of this method. Please remove the other calls, so that it is only called once.